lib/ncr/approval_manager.rb

```diff
@@ -0,0 +1,90 @@
+module Ncr
+  class ApprovalManager
+    attr_reader :work_order
+
+    delegate :proposal, to: :work_order
+
+    def initialize(work_order)
+      @work_order = work_order
+    end
+
+    def system_approver_emails
+      if %w(BA60 BA61).include?(self.work_order.expense_type)
+        ba_6x_approver_emails
+      else
+        [ba_80_approver_email]
+      end
+    end
+
+    def setup_approvals_and_observers
+      if self.work_order.emergency
+        setup_emergency
+      else
+        setup_non_emergency
+      end
+    end
+
+    protected
```

Comment:
I don't really see the value of `private` over `protected`, based on the reason [above](#discussion_r44192717). Strong feelings?